developer_tests/utilities/work/mkmf_parse_args_test

```diff
@@ -0,0 +1,10 @@
+#!/bin/csh
+#
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+
+     ../../../build_templates/mkmf -p parse_args_test \
+ -t  ../../../build_templates/mkmf.template \
+ -a "../../.." path_names_parse_args_test
```

Comment:
I will state that I have gone to this style of mkmf_ files to keep the line lengths short enough to show up well in side-by-side comparisons. ... just a comment ...